pipeline-stack.ts

```diff
@@ -81,6 +81,10 @@ export class PipelineStack<A> extends Stack {
         return [];
     }
 
+    getOtherPipelineProps(accountConfig: Account, pipelineConfig: Instance<A>, stackProps: StackProps) {
```

Comment:
This should have return-type `Partial<pipelines.CodePipelineProps>`, I think. I'm sad the linter let you get away without one.